<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramelessWindowCentralWidget</class>
 <widget class="QWidget" name="FramelessWindowCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleBarWidget {

	border-top-left-radius: 4px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));

	border-top-right-radius: 4px;
}
#titleBarWidget:hover {
	border-top-left-radius: 4px;

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.392045 rgba(198, 198, 198, 209), stop:0.693182 rgba(241, 241, 241, 255));
	border-top-right-radius: 4px;
}
#titleBarWidget #titleLabel {
	padding: 8px 6px 6px 12px;
	font: 75 9pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}
#titleBarWidget #titleLabel:hover{	
	
	color: rgb(0, 85, 255);
	font: 75 9pt &quot;微软雅黑&quot;;
}

#titleBarWidget .QPushButton {
	border: 0px solid red;
	border-radius: 8px;
	width: 16px;
	height: 16px;
	margin-right: 5px;
	background: rgb(78, 119, 255);
	color: white;
}

.FramelessWindowCentralWidget {
	border-radius: 4px; 
	background: white; 
	border: 1px solid lightgray;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="titleBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QWidget" name="buttonsWidget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="maximizeButton">
           <property name="text">
            <string>+</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="minimizeButton">
           <property name="text">
            <string>-</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>x</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>标题栏</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
